src/tests/employees.test.js

```diff
@@ -1,5 +1,280 @@
-describe('example', () => {
-  it('Test', () => {
-    expect(true).toEqual(true);
+import request from 'supertest';
+import app from '../app';
+import Employees from '../models/Employees';
+import employeesSeed from '../seeds/employees';
+
+describe('Time-sheet - Unit tests', () => {
+  beforeAll(async () => {
+    await Employees.collection.insertMany(employeesSeed);
+  });
+
+  describe('GET /employees', () => {
+    test('Should return status code 200', async () => {
+      const response = await request(app).get('/employees').send();
+      expect(response.status).toBe(200);
+      expect(response.body.error).toBeFalsy();
+      expect(response.body.data).toBeDefined();
+      expect(response.body.message).toBe('Employees found');
+    });
+  });
+
+  describe('GET BY ID /employees/:id', () => {
+    test('Should return status code 200', async () => {
+      const response = await request(app).get('/employees/634d5803354e41cd60b9e400').send();
+      expect(response.status).toBe(200);
+      expect(response.body.error).toBeFalsy();
+      expect(response.body.data).toBeDefined();
+      expect(response.body.message).toBe('Employee found');
+    });
+  });
+
+  describe('GET BY ID ERROR /employees/:id', () => {
+    test('Should return status code 404', async () => {
```

Comment:
Why the request is returning a status 404?
You have to add the purpose why this is going to return an error. For example:
```suggestion
    test('should return an status 400 when the employee does not exist in the database', async () => {
```